README.md

```diff
@@ -78,9 +78,9 @@ Minecraft uses resource packs to assign characters (glyphs) to different unicode
 |        Middle Mouse Button       |   Windows   |          |  U+E062 |         ![](./images/middle-mouse.png)         |
 |               Mouse              |   Windows   |          |  U+E063 |             ![](./images/mouse.png)            |
 |         Forward Arrow New        |    Mobile   |          |  U+E065 |        ![](./images/forward-arrow-new.png)     |
-|          Right Arrow New         |    Mobile   |          |  U+E066 |         ![](./images/right-arrow-new.png)      |
+|          Left Arrow New          |    Mobile   |          |  U+E066 |         ![](./images/left-arrow-new.png)       |
```

Comment:
Were left/right in the wrong positions before? Please confirm that the unicode is correct.